manifests/params.pp

```diff
@@ -6,6 +6,8 @@
 class virtualbox::params {
   $manage_ext_repo = true
   $repo_proxy      = undef
+  $apt_key_thumb   = undef
+  $apt_key_source  = undef
```

Comment:
:open_mouth: ok I will look at this today. I tested where it didn't work (centos) but not where it was working (debian).